Jantimizer/QueryTestSuite/Connectors/PostGres.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Npgsql;
```

Comment:
Could it make sense for this to called 'PostgresConnector' rather than just 'PostGres'? Moreover why is postgres capitalized if it were two words?